tests/phpunit/tests/block-supports/duotone.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * Test the block WP_Duotone class.
+ *
+ * @package WordPress
+ */
+
+class WP_Duotone_Test extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Block_Supports_DuoTones extends WP_UnitTestCase {
```

No WP prefix in test suite.